Gemfile

```diff
@@ -83,6 +85,7 @@ group :test do
   gem "selenium-webdriver"
   gem "simplecov", require: false
   gem "rails-controller-testing"
+  gem "mocha"
```

Comment:
I don't think there is a built-in way within mini test to mock things. Since the code is now making API calls I want to exercise most of the code but short circuit right where the API call is made. Mocha is the way I've done this in the past and it's always felt pretty clean, but I'm open to other suggetsions.

Reply:
Minitest has both [mocks](https://www.rubydoc.info/gems/minitest/Minitest/Mock#expect-instance_method) and [stubs](https://www.rubydoc.info/gems/minitest/Object#stub-instance_method). It's a minimal implementation, like the rest of minitest, but it works. I think minitest's vernacular for what a mock is and does differs a bit from the rspec world, and I always have to look it up.

I tend to think of Minitest's mocks/stubs as "syntactic salt" -- they're muddy to use, and I think they do it that way on purpose because "clean test code rarely uses mocks". 

I'm not opposed to including mocha, but maybe you won't feel the need after seeing what minitest has to offer. I made a single suggestion below with what I believe the stub would look like without mocha.

Reply:
I think I might amend what I said about salt to clarify that the mocks are salty, but I don't think the stubs are any more difficult to use than anything else out there.

Reply:
@robacarp Oh sheesh, chatgpt had implied I needed mocha and that my brain said "oh that's right, there is no stubbing in minitest" and I didn't actually check. :) Their syntax is good enough, I'm switching.